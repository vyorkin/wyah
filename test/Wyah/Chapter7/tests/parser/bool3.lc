let T x y = x;
let F x y = y;
