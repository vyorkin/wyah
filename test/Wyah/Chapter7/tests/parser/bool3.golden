Program [Decl "T" (ELam (Var "x") (ELam (Var "y") (EVar (Var "x")))),Decl "F" (ELam (Var "x") (ELam (Var "y") (EVar (Var "y"))))]